app/views/community_memberships/_confirmation_form.haml

```diff
@@ -2,11 +2,14 @@
   initialize_confirmation_pending_form("#{I18n.locale}","#{email_not_accepted_message}");
 
 - email_to_confirm = @current_user.latest_pending_email_address(@current_community)
+- contact_support_link = link_to t("sessions.confirmation_pending.contact_support_link_text"), "mailto:#{support_email}", "data-uv-trigger" => "contact"
```

Comment:
What is the data-uv-trigger for? I searched through the repo but didn't find full answer, but it seemed to be included in several places where support mailto links are used, so my guess was that it is related to tracking somehow? Anyway seemed better to follow the example and include it here, but of course would be good to know what it does, and not blindly follow examples. So please let me know what's the purpose, and is it right to include it here?